mungegithub/mungers/owner-approval.go

```diff
@@ -0,0 +1,225 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/contrib/mungegithub/github"
+	"k8s.io/kubernetes/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/util/yaml"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+)
+
+const (
+	ownerApproval = "has-owner-approval"
+)
+
+type assignmentConfig struct {
```

Comment:
if I interpret correctly, this is the type that the OWNERS file is going to unmarshal into, yes?

Can we make it be a struct so we have room to grow?

```
type OwnersFile struct {
    Owners []string `json:owners`
 }
```

This then leaves me room for adding maintainers or other metadata.